client/flutter/lib/pages/main_pages/home_page_router.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:who_app/constants.dart';
+import 'package:who_app/pages/main_pages/check_up_page.dart';
+import 'package:who_app/pages/main_pages/home_page.dart';
+import 'package:who_app/pages/main_pages/learn_page.dart';
+import 'package:who_app/pages/main_pages/settings_page.dart';
+import 'package:who_app/pages/main_pages/stats_page.dart';
+
+class HomePageRouter extends StatelessWidget {
```

Comment:
This is nitpicky and not a blocker for this PR, but we may want to consider naming this something that indicates it's the main tab-based router for the application as a whole, e.g. `MainTabRouter` / `AppTabRouter` or something along those lines.

The reason for this being: we may have a sub-router or navigation within the Home tab of the app, and that could get confusing to see multiple routers with Home in the name.